src/omniperf_profile/profiler_base.py

```diff
@@ -162,15 +169,15 @@ def join_prof(self, out=None):
         for key, cols in duplicate_cols.items():
             _df = df[cols]
             if not test_df_column_equality(_df):
-                msg = "WARNING: Detected differing {} values while joining pmc_perf.csv".format(
+                msg = "Detected differing {} values while joining pmc_perf.csv".format(
                     key
                 )
-                logging.warning(msg + "\n")
+                console_warning(msg + "\n")
             else:
                 msg = "Successfully joined {} in pmc_perf.csv".format(key)
-                logging.debug(msg + "\n")
+                console_debug(msg + "\n")
             if test_df_column_equality(_df) and self.__args.verbose:
-                logging.info(msg)
+                console_log("profile", msg)
```

Comment:
Everywhere else we've been using the "profiling" keyword, as opposed to "profile" in logging output. Consider changing for consistency
```suggestion
                console_log("profiling", msg)
```

Reply:
Good catch - thanks.